test/performance/odp_dma_perf.c

```diff
@@ -1123,6 +1124,21 @@ static void drain_compl_events(void)
 	}
 }
 
+static void
+drain_poll_transfers(sd_t *sd)
+{
+	const uint32_t count = sd->dma.num_inflight;
+	trs_info_t *infos = sd->dma.infos, *info;
+	odp_dma_t handle = sd->dma.handle;
+	odp_dma_result_t res;
+
+	for (uint32_t i = 0U; i < count; ++i) {
+		info = &infos[i];
+		if (info->is_running)
+			odp_dma_transfer_done(handle, info->compl_param.transfer_id, &res);
```

Comment:
Would the draining be more reliable if there was a `while` here? So, looping as long as `odp_dma_transfer_done()` returns 0 ("transfer has not finished").